app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User <ApplicationRecord
 
   belongs_to :merchant, optional: true
+  has_many :orders
```

Comment:
From new migration

Reply:
Glad we made this migration, It will help me with my user stories as well